CHANGELOG.md

```diff
@@ -1,5 +1,139 @@
+execution of the running script and then the associated script file will be opened
+in the editor so that you can set breakpoints and step through its execution.
+
+We've also added a new `launch.json` configuration for debugging PowerShell host processes:
+
+![Process launch configuration screenshot](https://cloud.githubusercontent.com/assets/79405/22089468/391e8120-dda0-11e6-950c-64f81b364c35.png)
+
+When launched, the default "attach" configuration will prompt you with a list of
+PowerShell host processes on the local machine so that you can easily select one
+to be debugged:
+
+![Process selection UI screenshot](https://cloud.githubusercontent.com/assets/79405/22081037/c205e516-dd76-11e6-834a-66f4c38e181d.png)
+
+You can also edit the launch configuration to hardcode the launch parameters, even
+setting a remote machine to connect to before attaching to the remote process:
+
+```json
+        {
+            "type": "PowerShell",
+            "request": "attach",
+            "name": "PowerShell Attach to Host Process",
+            "computerName": "my-remote-machine",
+            "processId": "12345",
+            "runspaceId": 1
+        }
+```
+
+Please note that we currently do not yet support initiating remote sessions from Linux
+or macOS.  This will be supported in an upcoming release.
+
+#### Initial support for remote file opening using `psedit`
+
+Another nice improvement is that we now support the `psedit` command in remote and
+attached sessions.  This command allows you to open a file in a local or remote session
+so that you can set breakpoints in it using the UI before launching it.  For now these
+remotely-opened files will not be saved back to the remote session when you edit and
+save them.  We plan to add this capability in the next feature update.
+
+#### New "interactive session" debugging mode
+
+You can now create a new launch configuration which drops you directly into the
+debug console so that you can debug your scripts and modules however you wish.
+You can call Set-PSBreakpoint to set any type of breakpoint and then invoke your
+code through the console to see those breakpoints get hit.  This mode can also be
+useful for debugging remote sessions.
+
+![Interactive session config screenshot](https://cloud.githubusercontent.com/assets/79405/22089502/5e56b4c6-dda0-11e6-8a51-f24e29ce7988.png)
+
+Please note that this is NOT a replacement for a true interactive console experience.
+We've added this debugging configuration to enable a few other debugging scenarios, like
+debugging PowerShell modules, while we work on a true interactive console experience using
+VS Code's Terminal interface.
+
+#### Other fixes and improvements
+
+- Added a new `Open Examples Folder` command to easily open the extension's
+  example script folder.
+- Added a new setting `powershell.developer.powerShellExeIsWindowsDevBuild`
+  which, when true, indicates that the `powerShellExePath` points to a Windows
+  PowerShell development build.
```

Comment:
I would suggest an addition here:

Added support for providing symbol outline for psd1 files.